zork++/src/lib/compiler/mod.rs

```diff
@@ -13,6 +12,7 @@ use crate::{
     compiler::commands::Commands,
     config_file::ZorkConfigFile,
     project_model::{
+        arguments::Argument,
```

Comment:
Mmmm... still doesn't agree with this moves. `Argument` is still a command line thing, that just wraps data from the models...